### PR TITLE
Feature/security plan

### PR DESCRIPTION
**Pull request by wandmagic**

### CODE REVIEW

File: .oscal/ssp.oscal.yaml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+          - pw2
+          - pw4
+          - pw5
+          - pw6
+          - pw7
+          - pw8
+          - pw9
+          - rv1
+          - rv2
+          - rv3
```
Comment (aj-stein-gsa) [inline .oscal/ssp.oscal.yaml:48]: Blocking: Can you do this in a SSP?

https://pages.nist.gov/OSCAL-Reference/models/v1.1.2/system-security-plan/json-outline/

File: .oscal/ssp.oscal.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
+    parties:
+      - uuid: 11111111-0000-4000-9000-100000000001
+        type: organization
+        name: "Metaschema Framework Orginization"
```
Comment (aj-stein-gsa) [inline .oscal/ssp.oscal.yaml:11]: Minor typo on org name:

```suggestion
        name: "Metaschema Framework Organization"
```

File: .oscal/ssp.oscal.yaml [hunk 3]
```diff
@@ -0,0 +1,162 @@
+system-security-plan:
+  uuid: 8c1ee2a6-bd14-4f77-b2f7-3d2bc0aaa55e
+  metadata:
+    title: "Oscal Server Software Development System Security Plan"
+    last-modified: "2024-10-24T00:00:00Z"
+    version: "1.0"
+    oscal-version: "1.0.4"
```
Comment (aj-stein-gsa) [inline .oscal/ssp.oscal.yaml:7]: Question: why 1.0.4 and not current stable (1.1.2)?

```suggestion
    oscal-version: "1.1.2"
```

File: .oscal/ssp.oscal.yaml [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+        title: System Owner
+      - id: security-officer 
+        title: Security Officer
+    responsible-parties:
+      - role-id: system-owner
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
+      - role-id: security-officer
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000001
```
Comment (aj-stein-gsa) [inline .oscal/ssp.oscal.yaml:24]: Re a previous conversation, we really need to meet to discuss what these roles and what they would mean. To sit, SSDF doesn't really need or explicate those, that's a RMF SP 800-53 concept.

Let's put a meeting on the books to discuss this.

File: .oscal/component.oscal.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      description: >
+        A service that runs through local loopback on port 8888
+      props:
+        - name: port
+          value: "8888"
+      protocols:
+        - name: https
+          port-ranges:
+            - start: 8888
+              end: 8888
```
Comment (aj-stein-gsa) [inline .oscal/component.oscal.yaml:21]: Blocking: why not use protocols?

https://pages.nist.gov/OSCAL-Reference/models/v1.1.2/component-definition/json-reference#/component-definition/components/protocols

File: .oscal/ssdf.catalog.oscal.yaml [hunk 6]
```diff
@@ -0,0 +1,1058 @@
...
+          acquisition,software development,software development life cycle
+          (SDLC),software security
+    roles:
+      - id: creator
+        title: Creator of the Original Content
+      - id: prepared-by
+        title: Author of Content in OSCAL Format
+    title: >-
+      Secure Software Development Framework (SSDF) Version 1.1: Recommendations
+      for Mitigating the Risk of Software Vulnerabilities
```
Comment (aj-stein-gsa) [inline .oscal/ssdf.catalog.oscal.yaml:701]: As discussed before can we meet with the group and discuss the larger picture and next steps? I think SSDF is a good choice but we should sync on what that means for us next and how this work plays out.

**Review by aj-stein-gsa — CHANGES_REQUESTED** [review]
This is an amazing start. I had some minor nits and higher level commentary, can we meet to discuss this next week with @david-waltermire? Really solid start!

File: .oscal/ssp.oscal.yaml (hunk 3 above)
Comment (wandmagic) [reply .oscal/ssp.oscal.yaml:7]: fixed

**Comment by aj-stein-gsa** [comment]
Will #44 obsolete this PR?

**Comment by wandmagic** [comment]
> Will #44 obsolete this PR?

yes

**Pull request closed without merging**